content/guides/accessibility/color-considerations.mdx

```diff
@@ -0,0 +1,144 @@
+---
+title: Color considerations
+description: Color is an important aspect of creating accessible, inclusive experiences. 
+---
+
+## Overview
+
+This guidance covers considerations that can't automatically be integrated into Primer. It supplements [accessibility guidance given in the Color portion of our Foundations documentation](https://primer.style/design/foundations/color#accessibility).
+
+### Design tokens
+
+The majority of color contrast considerations should be handled by [Primer color design tokens](https://primer.style/design/foundations/color#how-to-use-color-for-product-ui). Ensure that solutions for the following considerations integrate design tokens as a method of creating accessible experiences.
+
+### Determining contrast
+
+[WebAIM's Contrast Checker tool](https://webaim.org/resources/contrastchecker/) is the recommended way to calculate color contrast ratios.
+
+### The APCA
+
+The [Advanced Accessible Perceptual Contrast Algorithm](https://www.myndex.com/APCA/) (<abbr>APCA</abbr>) is a proposal and is **not normative**. In terms of WCAG, and therefore, MAS-C compliance, this means that color contrast is valid **only** if it meets WCAG 2.1 criteria. 
+
+A color value that is APCA-compliant **may not** be WCAG compliant until APCA is integrated into WCAG 3.0. WCAG 3.0 is still a work in progress and currently does not have an anticipated release date.
+
+## Compliance 
+
+The following guidance is based on one or more **normative** Web Content Accessibility Guidelines (<abbr>WCAG</abbr>) success criterion. This means that it is objective requirements with implications that affect MAS-C compliance efforts, as well as international laws regarding accessibility.
+
+Non-normative suggestions are also supplied, but marked as such.
+
+## Visual themes
+
+GitHub offers two parent themes: Day (light) and Night (dark). Each parent theme has a default appearance, and then a suite of sub-themes. Sub-themes include considerations for high and low contrast experiences, as well as certain vision conditions.
+
+It is possible to set GitHub to use a dark sub-theme when a parent Day theme has been set, and vice-versa. This allows a user to have a fine-grained control over their experience.
+
+### Default theme
+
+GitHub uses the default Day (light) theme when a user visits the site in a logged out state. Because the visitor is logged out, they are unable to adjust the theme in the [Appearance settings](https://github.com/settings/appearance).
+
+Because of this, [the default Day theme **must** to satisfy WCAG criteria for color-related concerns](https://github.com/github/accessibility/blob/main/docs/coaching-recommendations/primer/color-modes.md#guidance). From a high-level, this means that **all** GitHub content displayed using the default Day theme must:
```

Comment:
```suggestion
Because of this, [the default Day theme **must** satisfy WCAG criteria for color-related concerns](https://github.com/github/accessibility/blob/main/docs/coaching-recommendations/primer/color-modes.md#guidance). From a high-level, this means that **all** GitHub content displayed using the default Day theme must:
```

Reply:
Oh! Nice catch, thank you!